dts/msm8916/msm8916-vivo-y21l.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+    qcom,msm-id = <0xce 0x00 0xf8 0x00 0xf9 0x00 0xfa 0x00>;
+    qcom,board-id = <0x08 0x0d>;
```

Comment:
I am not sure if there will be another device with identical msm-id/board-id, since it's a MTP device `<8 *>`.
Try to add something such as `lk2nd,match-panel;` to distinguish them would be better.